code/src/pages/GameScreen.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components'
```

Comment:
if you write 'styled-components/macro' the macro part makes it easier to see in the console what div, container, etc. you are or were styling.. 🤩